generate/generate.go

```diff
@@ -444,6 +444,12 @@ func (g *Generator) SetProcessUsername(username string) {
 	g.Config.Process.User.Username = username
 }
 
+// SetProcessUmask sets g.Config.Process.User.Umask.
+func (g *Generator) SetProcessUmask(umask uint32) {
+	g.initConfigProcess()
+	g.Config.Process.User.Umask = umask
+}
```

Comment:
This has changed to be an `*uint32` with https://github.com/opencontainers/runtime-spec/commit/d3f079a3fd5c503631012f71bc0fda66dcc9e755, which breaks compatibility between the tools and the spec of both latest master branches.

Reply:
Just found the same thing :)

Reply:
So, there is a PR to fix it (https://github.com/opencontainers/runtime-tools/pull/708) but apparently it requires a new release of runtime-specs, for which there is an issue opened (https://github.com/opencontainers/runtime-spec/issues/1052) but it's not yet done so...